docs/development/install-prerequisites.md

```diff
@@ -0,0 +1,115 @@
+
+The K8S proxy calls service catalog APIs for service broker management and requires a K8S cluster with service catalog installed.
+
+**Note:** The Service Manager itself does not talk to the platforms , so technically if you want to run just the Service Manager, you may do so without any OSB-compliant platforms provided that you have a place to actually deploy SM.
+
+## Kubernetes Deployment Prerequisites
+
+Additional prerequisites to the [general deployment prerequisites](#general-deployment-prerequisites) required for installing the Service Manager (and/or the K8S Proxy) on a K8S Cluster.
+
+### kubectl
+
+Setup `kubectl` as described [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/). Also install the [svcat plugin](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install.md#plugin).
+
+### helm
+
+Setup `helm` on your K8S Cluster. Details about installing `helm` can be found [here](https://github.com/kubernetes/helm/blob/master/docs/install.md).
+
+Alternatively, **Windows** users may follow [these steps](https://medium.com/@JockDaRock/take-the-helm-with-kubernetes-on-windows-c2cd4373104b):
+
+
+Afterwards, start Minikube and initialize Tiller.
+
+```console
+$ minikube start --extra-config=apiserver.Authorization.Mode=RBAC
+
+$ helm init
+```
+
+ ### service catalog
+
+ Setup `service catalog` on your K8S Cluster. Details about installing the `service catalog` can be found [here](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/install.md).
+
+Alternatively, **Windows** users may follow these steps:
+
+ ```console
+ $ kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+
+ $ helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+
+ $ kubectl create clusterrolebinding tiller-cluster-admin  --clusterrole=cluster-admin --serviceaccount=kube-system:default
+
+ $ kubectl -n kube-system patch deployment tiller-deploy -p '{"spec": {"template": {"spec": {"automountServiceAccountToken": true}}}}'
+
+ $ helm install svc-cat/catalog  --name catalog --namespace catalog
+ ```
+
+## Cloud Foundry Deployment Prerequisites
+
+Additional prerequisites to the [general deployment prerequisites](#general-deployment-prerequisites) required for installing the Service Manager (and/or the CF Proxy) on CF.
+
+#### CF CLI
+
+Details about installing `CF CLI` can be found [here](https://github.com/cloudfoundry/cli#downloads).
+
+**Note:**  kubectl, svcat, helm and smctl binaries should be in your $PATH.
+
+#### Cloud Foundry Installation
+
+You would also need a Cloud Foundry installation and either user/pass or oauth client credentials to manage service brokers as well as service access at the Cloud Controller API.
+
+**Note:** The CF installation should have a `go_buildpack` installed with support for Go 1.10+. The buildpack should use the name `go_buildpack`. Steps:
```

Comment:
I think the steps can be removed. Can be added that at least version 1.8.19 https://github.com/cloudfoundry/go-buildpack/releases is required. (We might remove this at all, because we can adopt new go versions only after the cf buildpack has a version that works with this version)

Reply:
I'd rather keep it in one place - I removed it from the pcfdev section in develop-prerequisites but the install-prerequisites are kind of more general - I restructured the sentence a bit but I'd rather  keep it to be informative